bastionzero/service/agents/agenttype.go

```diff
@@ -0,0 +1,11 @@
+package agents
+
+// Where the agent is running. Either a kube cluster, a linux server, or a windows server
+type AgentType string
```

Comment:
Please follow the same comment I left for `agentstatus.go` and apply it here as well